.github/workflows/integration_tests.yaml

```diff
@@ -28,6 +30,6 @@ jobs:
 
             -   run: composer install --no-progress
             -   run: |
-                    bin/console doctrine:database:create --env=test
-                    bin/console doctrine:schema:create --env=test
+                    # We should run migrations instead, because we can check afterwards if migrations are up to date
+                    bin/console doctrine:schema:create
```

Comment:
How is the database created now?

Reply:
https://github.com/pehapkari/pehapkari.cz/blob/master/.docker/docker-entrypoint.sh#L26

It is not handled very well, we should change it to migrations in future :-)